src/view/ThemeSettings.js

```diff
@@ -100,7 +100,24 @@ define(function (require, exports, module) {
                 var attr = $target.attr("data-target");
                 newSettings[attr] = $target.is(":checked");
             })
-            .on("input", "[data-target]:text", function () {
+            .on("input", "#font-size-input", function () {
+                var $target = $(this);
+                var $input = $target.val();
```

Comment:
`$` prefix for variables indicates that it's a jquery (or a DOM) element
in this case `$input` is a string and should really be named `input` or even better `targetValue`